### PR TITLE
Hod/mimecast

### PR DESCRIPTION
**Pull request by hod-alpert**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://github.com/demisto/etc/issues/26906

## Description
* Removed branches from the test playbook
* Added conditions that will skip validating messages in case no messages are returned.
* Fixed an issue with downloaded attachments where in case of an error- the integration was just writing the error response as an attachment

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [x] No

### CODE REVIEW

**Comment by kirbles19** [comment]
@hod-alpert Done.

File: Packs/Mimecast/Integrations/MimecastV2/MimecastV2.py [hunk 1]
```diff
@@ -1647,8 +1647,12 @@ def download_attachment_request(attachment_id):
...
 
     response = http_request('POST', api_endpoint, str(payload), is_file=True)
-    if isinstance(response, dict) and response.get('fail'):
-        return_error(json.dumps(response.get('fail', [{}])[0].get('errors')))
+    try:
+        json_response = response.json()
+        if json_response.get('fail'):
+            return_error(json_response.get('fail', [{}])[0].get('errors'))
+    except ValueError:
+        pass
```
Comment (Itay4) [inline Packs/Mimecast/Integrations/MimecastV2/MimecastV2.py:1655]: lets log here the error

File: Packs/Mimecast/TestPlaybooks/playbook-Mimecast_Test.yml [hunk 2]
```diff
@@ -583,82 +580,13 @@ tasks:
...
-    quietmode: 0
-  "19":
-    id: "19"
-    taskid: af5fbc97-e57a-4703-82c6-20d8da662c07
-    type: regular
-    task:
-      id: af5fbc97-e57a-4703-82c6-20d8da662c07
-      version: -1
-      name: mimecast-get-message
-      script: '|||mimecast-get-message'
```
Comment (Itay4) [inline Packs/Mimecast/TestPlaybooks/playbook-Mimecast_Test.yml:608]: so we're not testing that command anymore?

File: Packs/Mimecast/Integrations/MimecastV2/MimecastV2.py (hunk 1 above)
Comment (hod-alpert) [reply Packs/Mimecast/Integrations/MimecastV2/MimecastV2.py:1655]: Just missed it, the doc says:
```
On success, the content of the file is returned as a stream.

On failure, the standard JSON API response is returned.
```
Do you want me open another PR about this?

Comment (Itay4) [reply Packs/Mimecast/Integrations/MimecastV2/MimecastV2.py:1655]: no need

**Pull request merged**